Recreated Kubesimplify README.md (#80)
kubesimplify/website: README.md

<h1 align="center"><a href="https://kubesimplify.github.io/website/">Kubesimplify Website </a></h1>
<p align="center"><i>The source code for Kubesimplify.com</i></p>
Kubesimplify's official website. This codebase is what runs on [Kubesimplify](https://kubesimplify.github.io/website/). 

This repository contains the assets required to build the Kubesimplify website and documentation.

## Contribute
We're glad that you want to contribute! 

> A first step to contributing is to start with something you think you can add value in te Community. Start attending Kubesimplify meetings, join the [Discord channels](https://discord.gg/uFBC8MyeeU) and subscribe to the [official youtube Channel](https://www.youtube.com/c/saiyam911). Ask in community, We always Welcome new Contributors.

Reviewer: ```suggestion
A first step to contributing is to start with something you think you can add value in the community. Start attending Kubesimplify meetings, join the [Discord channels](https://discord.gg/uFBC8MyeeU) and subscribe to the [official youtube Channel](https://www.youtube.com/c/saiyam911). Ask in community, We always Welcome new Contributors.
```

Reviewer: @pavangudiwada are we suppose to add this para in the readme as of now or maybe can you rephrase this to something that we should say to community folks who are not the ambassadors

> The Contributor Guide provides detailed instruction on how to get your ideas and bug fixes seen and accepted, including:

Reviewer: ```suggestion
```

> - [Code of Conduct](code-of-conduct.md)

Reviewer: ```suggestion
```

### How to file an issue 
Not ready to contribute code, but see something that needs work? While the community encourages everyone to contribute code, it is also appreciated when someone reports an issue.
Make sure to adhere to the prompted submission guidelines while opening an issue. 

### How to find something to work on
Kubesimplify is Fresh starting Community so there is a lot to do. Be in touch with [Discord Server](https://discord.gg/uFBC8MyeeU), Twitter and [official youtube Channel](https://www.youtube.com/c/saiyam911). There are a lot of ongoing projects and you can contribute to them.  

> ### How to open a pull request

Reviewer: ```suggestion

### How to open a pull request
```

We recommend open an issue before editing and Pulling request. If you don't know "How to open a Pull request" We refer you to [this resource](https://opensource.com/article/19/7/create-pull-request-github)

## Prerequisites

> To use this repository and to Run Kubesimplify's website on your computer, you need the installed dependencies locally:

Reviewer: ```suggestion
To use this repository and to Run Kubesimplify's website on your local device, follow the below steps:
```

> Before you start, install the dependencies. Clone the repository and navigate to the directory:

Reviewer: ```suggestion
```

Clone the repo to your local device: 

```bash
   git clone https://kubesimplify.github.io/website/ 
   cd website
```
Install [npm](https://www.npmjs.com/)

```bash
    $ npm install 
```

```bash
    $ npm start
```

And then open up your web browser and go to [localhost:3000](http://localhost:3000).

## Find something to work on

> The first step to getting starting contributing to Kubesimplify is to find something to work on. Help is always welcome, and no contribution is too small!

Reviewer: ```suggestion
The first step to getting started contributing to Kubesimplify is to find something to work on. Help is always welcome, and no contribution is too small!
```

> Become a part of Kubesimplify community by Contribute. All types of contribution are allowed (code contribution, Docs contribution and community part too)

Reviewer: ```suggestion
Become a part of the Kubesimplify community by contributing. All types of contributions are allowed (code contributions, docs contributions, and community contributions too).
```

Here are some things you can do today to get started contributing:

1. Help improve the Kubesimplify documentation
2. Clarify code, variables, or functions that can be renamed or commented on
3. Write test coverage
4. Help triage issues

## Community Code of Conduct

> We follow Code of conduct here. Participation in the Kubesimplify community is governed by the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)

Reviewer: Should this be CNCF code of conduct @saiyam1814 @anubha-v-ardhan

Author: @AvineshTripathi originally It was added as CNCF code of conduct.

### For new developers getting started, here are some instructions to follow:
1. Follow code of conduct 
2. Be respectful to every contributer, We all are working as a team.
3. Always open to new ideas and implementation.

## Membership

> We encourage all contributors to become members. We aim to grow an active, healthy community of contributors, reviewers, and code owners. Learn more about requirements and responsibilities of membership on our [Discord channels](https://discord.gg/uFBC8MyeeU).

Reviewer: I am not sure if we need this

Author: On this moment, of course, we don't need this. But soon we will in long term
